1-exercises/A-undefined/exercise.js

```diff
@@ -13,6 +13,8 @@
 let a;
```

Comment:
@DelroyGayle overall well-done for your hard work and doing the extras, my comment is just for you to be a better developer in future :) 
Happy coding 🎊

Reply:
Thank you for your comments Ellie. I have made changes accordingly to the best of my understanding. Thank you